index.js

```diff
@@ -63,7 +60,17 @@ module.exports = options => {
 	options = options || {};
 
 	if (process.platform === 'win32') {
-		return [];
+		const histFile = getPath(options);
```

Comment:
The naming is an implementation detail of the `getPath` method.
```suggestion
		const historyPath = getPath(options);
```